src/content/api/agent/v1.1/requests/preupdate.apib

```diff
@@ -0,0 +1,93 @@
+The request body must be a JSON object with exactly 4 properties, includes deployment_name, instance_guid, sf_operations_args, params.
+
+    + Body
+
+            {
+                "deployment_name": "service-fabrik-1234-12345678-1234-5678-1234-123456781234",
+                "instance_guid": "12345678-1234-5678-1234-123456781234",
+                "params": {
+                    "context": {
+                        "organization_guid": "12345678-1234-5678-1234-123456781234",
+                        "platform": "cloudfoundry",
+                        "space_guid": "12345678-1234-5678-1234-123456781234"
+                    },
+                    "previous_values": {
+                        plan_id: '12345678-1234-5678-1234-123456781234',
+                        service_id: '12345678-1234-5678-1234-123456781234',
+                        organization_id: '12345678-1234-5678-1234-123456781234',
+                        space_id: '12345678-1234-5678-1234-1234567812341'
+                    },
+                    "plan_id": "12345678-1234-5678-1234-123456781234",
+                    "service_id": "12345678-1234-5678-1234-123456781234",
+                    "previous_manifest": {
+                        ...
+                    }
```

Comment:
also include some command line args input as part of cf update-service operation.